### PR TITLE
Remove (package.json).files

### PR DESCRIPTION
**Pull request by jsumners**
This addresses what was left unfinished from https://github.com/pinojs/pino/pull/1588 and hopefully makes it clear we want the files in the package (re: https://github.com/pinojs/pino/issues/1672).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,19 +6,6 @@
   "type": "commonjs",
   "types": "pino.d.ts",
   "browser": "./browser.js",
-  "files": [
```
[the remark this answers is not in the document]
Comment (jsumners) [reply package.json:9]: > I'm afraid removing this much would result in `test` folder still be published.

Yes. That is correct.

[the remark this answers is not in the document]
Comment (koresar) [reply package.json:9]: Any explanation why people want `/test/` files uploaded to npm (and then to millions of computers world wide) ?

[the remark this answers is not in the document]
Comment (jsumners) [reply package.json:9]: So that they can be referenced when navigating a project's source tree.

**Comment by kibertoad** [comment]
I don't see any benefits in this change.

**Comment by koresar** [comment]
@kibertoad would you see benefits in removing the "test" folder (only!) from the npm tarball?

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (koresar) [reply package.json:9]: After `git checkout` anyone can navigate the source tree.

Why do we upload the "test" fodler to NPM?
https://www.npmjs.com/package/pino?activeTab=code

**Comment by jsumners** [comment]
First, how are these a degradation of your experience? Second, a link in a readme isn't going to work when an internet connection is not available or is very unreliable.

**Comment by Fdawgs** [comment]
> @koresar I do, but I think I'm in the minority in the pino team. There was this discussion before, and broadly pino (and fastify) teams consider tests to be part of project documentation, and prefer it bundled for easy perusal by consumers of the libraries.

See https://github.com/fastify/skeleton/issues/42

Not a fan of unnecessary files being added either from both an environmental and security perspective.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**